homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -31,6 +31,8 @@ class BMWDataUpdateCoordinator(DataUpdateCoordinator[None]):
     """Class to manage fetching BMW data."""
 
     account: MyBMWAccount
+    read_only: bool
+    config_entry: ConfigEntry
 
     def __init__(self, hass: HomeAssistant, *, entry: ConfigEntry) -> None:
```

Comment:
```suggestion
    def __init__(self, hass: HomeAssistant, *, config_entry: ConfigEntry) -> None:
```
For consistency in the same file at least :)